apps/admin/traefik2/prod/secret-provider.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: secrets-store.csi.x-k8s.io/v1alpha1
+kind: SecretProviderClass
+metadata:
+  name: traefik-default-cert
+  namespace: admin
+spec:
+  parameters:
+    keyvaultName: acmedcdcftappsprod
+    objects: |
+      array:
+        - |
+          objectName: wildcard-prod-platform-hmcts-net
```

Comment:
Naming is not correct to what is reflective in KV

Reply:
Also is internal cert needed for prod? @hannah38

Reply:
[> ](https://github.com/hmcts/cnp-flux-config/pull/17026#discussion_r936382677)

Resolved this, corrected the name to match wildcard from the kv.